commit adb25b631feb2ac3fd0126e954bf988c85656c5c
Author: Andreas Sturmlechner <andreas.sturmlechner@gmail.com>
Date:   Thu Nov 24 20:17:33 2016 +0100

    find_dependency takes one dependency per call
    
    Fixes bogus 'target KF5::Service not found' in reverse deps
    like kipi-plugins.
    
    REVIEW: 129555

diff --git a/cmake/templates/KF5KipiConfig.cmake.in b/cmake/templates/KF5KipiConfig.cmake.in
index 0bd3ba9..4f92b62 100644
--- a/cmake/templates/KF5KipiConfig.cmake.in
+++ b/cmake/templates/KF5KipiConfig.cmake.in
@@ -13,7 +13,11 @@
 
 @PACKAGE_INIT@
 
-find_dependency(Qt5Core Qt5Widgets Qt5Gui @REQUIRED_QT_VERSION@)
-find_dependency(KF5Config KF5XmlGui KF5Service)
+find_dependency(Qt5Core @REQUIRED_QT_VERSION@)
+find_dependency(Qt5Gui @REQUIRED_QT_VERSION@)
+find_dependency(Qt5Widgets @REQUIRED_QT_VERSION@)
+find_dependency(KF5Config)
+find_dependency(KF5Service)
+find_dependency(KF5XmlGui)
 
 include("${CMAKE_CURRENT_LIST_DIR}/KF5KipiTargets.cmake")
